modules/core/src/test/java/org/apache/ignite/internal/future/InFlightFuturesTest.java

```diff
@@ -40,6 +40,16 @@ void addsFutureToInFlightSetOnRegistration() {
         assertThat(currentFutures(inFlightFutures), is(singleton(incompleteFuture)));
     }
 
+
+    @Test
+    void addsAlreadyCompletedFutureToInFlightSet() {
```

Comment:
The idea of the test seems to be to check that the future actually gets removed (not added). How about renaming the test method to `removesAlreadyCompletedFutureOnRegistration()`?

Reply:
Yes, this makes sense to me. Will change the name.